### PR TITLE
New machines marked as NOT_WORKING

### PR DESCRIPTION
**Pull request by grullosgo**
New machines marked as NOT_WORKING 
---------------------------------- 
Channel Three [Team Europe, Grull Osgo]

### CODE REVIEW

File: src/mame/drivers/goldstar.cpp [hunk 1]
```diff
@@ -17671,6 +17712,38 @@ void wingco_state::init_luckylad()
...
+void cmaster_state::init_chthree()
+{
+	// Address swapping in 0x10 bytes blocks.
+	// 01234567-89abcdef <> c9ba8dfe - 41320576
+ 
+	uint8_t *ROM = memregion("maincpu")->base();
+	uint8_t row; 
+	uint16_t addr;
+
+	for (int A = 0000; A < 0x8000; A++)
```
Comment (cuavas) [inline src/mame/drivers/goldstar.cpp]: Please use `snake_case` for local variables, following the conventions used throughout the codebase.

File: src/mame/drivers/goldstar.cpp [hunk 2]
```diff
@@ -17671,6 +17712,38 @@ void wingco_state::init_luckylad()
...
+			case 0x03:  addr = 0x0a;break;
+			case 0x04:  addr = 0x08;break;
+			case 0x05:  addr = 0x0d;break;
+			case 0x06:  addr = 0x0f;break;
+			case 0x07:  addr = 0x0e;break;
+		}
+		addr = (A & 0xfff0) | addr;
+		if(!BIT(A,3)) std::swap(ROM[A], ROM[addr]);
+	}
+	// Temporal hacks - PPI settings and checksum correction
```
Comment (cuavas) [inline src/mame/drivers/goldstar.cpp]: Temporal and temporary are not synonyms.

File: src/mame/drivers/goldstar.cpp [hunk 3]
```diff
@@ -17671,6 +17712,38 @@ void wingco_state::init_luckylad()
...
+			case 0x01:  addr = 0x09;break;
+			case 0x02:  addr = 0x0b;break;
+			case 0x03:  addr = 0x0a;break;
+			case 0x04:  addr = 0x08;break;
+			case 0x05:  addr = 0x0d;break;
+			case 0x06:  addr = 0x0f;break;
+			case 0x07:  addr = 0x0e;break;
+		}
+		addr = (A & 0xfff0) | addr;
+		if(!BIT(A,3)) std::swap(ROM[A], ROM[addr]);
```
Comment (cuavas) [inline src/mame/drivers/goldstar.cpp]: Please use consistent spacing with flow control constructs.

File: src/mame/drivers/goldstar.cpp [hunk 4]
```diff
@@ -17671,6 +17712,38 @@ void wingco_state::init_luckylad()
...
 }
 
+void cmaster_state::init_chthree()
+{
+	// Address swapping in 0x10 bytes blocks.
+	// 01234567-89abcdef <> c9ba8dfe - 41320576
+ 
+	uint8_t *ROM = memregion("maincpu")->base();
+	uint8_t row; 
+	uint16_t addr;
```
Comment (cuavas) [inline src/mame/drivers/goldstar.cpp]: Please scope these variables appropriately, and use `const` where it makes sense.

File: src/mame/drivers/goldstar.cpp [hunk 5]
```diff
@@ -17671,6 +17712,38 @@ void wingco_state::init_luckylad()
...
+		switch (row)
+		{
+			case 0x00:  addr = 0x0c;break;
+			case 0x01:  addr = 0x09;break;
+			case 0x02:  addr = 0x0b;break;
+			case 0x03:  addr = 0x0a;break;
+			case 0x04:  addr = 0x08;break;
+			case 0x05:  addr = 0x0d;break;
+			case 0x06:  addr = 0x0f;break;
+			case 0x07:  addr = 0x0e;break;
```
Comment (cuavas) [inline src/mame/drivers/goldstar.cpp]: Please don't jam statements together with no space between them at all.

File: src/mame/drivers/goldstar.cpp (hunk 1 above)
Comment (grullosgo) [reply src/mame/drivers/goldstar.cpp]: There are a lot of lines using the same "A" variable into this driver.....must I edit all the wrong lines?

File: src/mame/drivers/goldstar.cpp (hunk 2 above)
Comment (grullosgo) [reply src/mame/drivers/goldstar.cpp]: ok.

File: src/mame/drivers/goldstar.cpp (hunk 1 above)
Comment (cuavas) [reply src/mame/drivers/goldstar.cpp]: You can change them all now if you want or leave that for another time – I just don't want any more added.  We use `SCREAMING_SNAKE_CASE` for enumerated constants, preprocessor macros and certain globals that behave like constants (mostly `constexpr` globals and device types), `LlamaCase` for template parameters, and `snake_case` for pretty much everything else.

Comment (grullosgo) [reply src/mame/drivers/goldstar.cpp]: ok.

**Pull request closed without merging**